tests/MongoDB.Driver.Tests/MongoCollectionImplTests.cs

```diff
@@ -258,6 +258,46 @@ public void Settings_should_be_set()
             findOperation.Sort.Should().BeNull();
         }
 
+        [Theory]
+        [ParameterAttributeData]
+        public void Aggregate_should_recognize_short_merge_collection_argument(
```

Comment:
Did you mean "string" instead of "short"?

Reply:
Should you add tests for the other two forms of the $merge arguments? I don't think there are any.

This one test could probably test all 3 scenarios with the right arguments.

Reply:
> Did you mean "string" instead of "short"?

I meant the short (or simplified) form (on contrary to full form: `{ $merge : { into : "collection" } }`).

> This one test could probably test all 3 scenarios with the right arguments.

Done.